BuildBom/SolutionGenerator.cs

```diff
@@ -137,5 +166,27 @@ public override bool Execute()
 
             return projectDataMap;
         }
+
+        private void GenerateMergedFile(List<BdioNode> components)
```

Comment:
The C# convention is that public methods and properties start with an uppercase letter. Private methods and properties do not.